skaffold.yaml

```diff
@@ -1,5 +1,5 @@
 .aea_version: &aea_version
-  AEA_VERSION: "1.21.0"
+  AEA_VERSION: "1.22.0.post1"
```

Comment:
No here it should not be post1 I think. Or are we using this for actually pulling the dependency?

Reply:
We need it for pulling the dependency